packages/io/stl-deserializer/src/index.js

```diff
@@ -410,9 +410,9 @@ const solid${index} = () => {
   return src
 }
 
-const extension = 'stl'
+const mimeType = 'application/sla'
```

Comment:
Hm, mimeType is important, but having extension available conceptually is not bad, to allow for automatic addition of extension when exporting to a file (with save dialog). Is extension info removed, or just moved elsewhere ?

Reply:
@hrgdavor excellent question.

so, the deserializers/serializers work on a specific file format, which is designated by the 'mimeType'. it's kind of a standard.

the file extensions can be found in the supportedFormats, and there are functions to look up formats for input. output, etc.

there's a new function called getMimeType(extension). would the opposite be useful? getExtension(mimeType)?

Reply:
getExtension(mimeType) would be useful, but it is not mission critical. It is not a big deal when setting up download menu in UI to define myself for each serializer->extension. But a thing like this would maybe allow more generic approach.